cloudapp/src/app/main/main.component.ts

```diff
@@ -20,7 +20,7 @@ export interface Criteria {
 export class MainComponent implements OnInit, OnDestroy {
   
   private pageLoad$: Subscription;
-  base_url: string;
+  base_url: string = "";
```

Comment:
This was causing an error when the app is loaded before the base_url is initialized